### ISSUE TITLE
IEC 74xx symbols

### ISSUE THREAD

**Issue by mrnuke** [KiCad/kicad-library#1826]
I'm working on IEC symbols for the 7400 series [here](https://github.com/mrnuke/kicad_iec_symbols).

I'm trying to figure out how to integrate this the existing library. In the master branch, I only have those symbols which have been production tested. i.e. the schematic was laid out, a board was manufactured, assembled and verified to be working.

What's the best way to go about incorporating this?

**Comment by SchrodingersGat** [comment]
See notes in your PR :)

**Comment by bobc** [comment]
I did quite a lot of work on my scripts to generate IEC style symbols, but had not quite finished. As the scripts became larger I had to do some essential refactoring.

I started with cmos_ieee because there were fewer symbols in that lib. A typical example of script generated symbol:

![image](https://user-images.githubusercontent.com/972103/32772587-34160152-c91e-11e7-8c8e-ec25dd58ccdd.png)

The script generates control block, elements, can also handle grouped inputs. There is some automatic pin decoration of tristate pins, also ability to label a pin, a group or an element. It turns out most of the required special symbols are in the Unicode font already.

Here is a draft of a complete set.

[demo_cmos_ieee.pdf](https://github.com/KiCad/kicad-library/files/1470166/demo_cmos_ieee.pdf)
